common/src/main/proto/TSOProto.proto

```diff
@@ -49,7 +49,6 @@ message CommitResponse {
     optional bool aborted = 1;
     optional int64 startTimestamp = 2;
     optional int64 commitTimestamp = 3;
-    optional bool makeHeuristicDecision = 4 [default = false];
```

Comment:
it's considered a bad habbit editing protos, better move down, comment out and mark deprecated


message CommitResponse {
    optional bool aborted = 1;
    optional int64 startTimestamp = 2;
    optional int64 commitTimestamp = 3;


// ============ deprecated
//    optional bool makeHeuristicDecision = 4 [default = false];
}